Add host option (#234)
nimbella/command-sets: github/README.md

In order to use this command set, you need to set up a Personal Access Token on GitHub with repo access. And save the token as a secret with `github_token` as the key name. Use `/nc secret_create` to create secrets.

Generate a token with `repo` access [here](https://github.com/settings/tokens) or click [here](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) to learn how to create a Personal Access Token on GitHub.

Create a secret named `github_repos` to avoid passing repository (`-r`) to commands.

![Secret Creator](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/secret_creator.png)

For commands that only work with single repository the first repository in `github_repos` is used.

### Github Enterprise

Only for Github Enterprise version, you need to specify the host. Below are the ways to do it, you can follow any of them
- Specify `--host` or `-h` while invoking the command.

> - Append `github_token` secret value with host name, followed by `@`, i.e. <your_access_token>@<your_github_host>.

Reviewer: ```suggestion
- Append `github_token` secret value with host name, followed by `@`, i.e. <your_access_token>@<your_github_host>. For example `mysecret@example.com`.

```

In order to use this command set, you need to set up a Personal Access Token on GitHub with repo access. And save the token as a secret with `github_token` as the key name. Use `/nc secret_create` to create secrets.

Generate a token with `repo` access [here](https://github.com/settings/tokens) or click [here](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) to learn how to create a Personal Access Token on GitHub.

Create a secret named `github_repos` to avoid passing repository (`-r`) to commands.

![Secret Creator](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/secret_creator.png)

For commands that only work with single repository the first repository in `github_repos` is used.

### Github Enterprise

Only for Github Enterprise version, you need to specify the host. Below are the ways to do it, you can follow any of them
- Specify `--host` or `-h` while invoking the command.

> - Append `github_token` secret value with host name, followed by `@`, i.e. <your_access_token>@<your_github_host>. For example `mysecret@example.com`.

Reviewer: ```suggestion
- Append the host name to your `github_token` following this format: `access_token@github_host`. For example `mysecret@example.com`.

```

> - Create secret with key name `github_host` using `/nc secret_create`, add host as value.

Reviewer: ```suggestion
- Create a secret with key name `github_host` using `/nc secret_create`, and add the host as the value.

```

> If you follow multiple ways, priority is decided top-down. 

Reviewer: ```suggestion
If you use more than one of the approaches listed above, the precedence is decided top-down (e.g., command line supersedes all others).
```

![GitHub create_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/create_issue.png)

> ### [`github_find_pr`](https://github.com/nimbella/command-sets/blob/master/github/packages/github/github_find_pr.js)

Reviewer: why is this removed?

Author: It had similar functionality as [github_view_pr.js](https://github.com/nimbella/command-sets/blob/master/github/packages/github/github_view_pr.js) and the actual implementation file (github_find_pr.js) was removed long back, only the document was not updated.
